Adds  ✨ hatch-fancy-pypi-readme + example

benjaoming opened the pull request.

benjaoming commented:
Making a change on request of the library owner..

Robson commented:
@benjaoming Thank you for making one of the changes. Just to confirm, do you see that there are more changes required than this?

I would also recommend splitting this into two pull requests:

- I would potentially be happy to approve the readme, if the changes are completed and I've had a more thorough review of it.
- I don't have the skills to understand the tool though (and all the emojis make me very hesitant, because it doesn't match the other descriptions), so I wouldn't be able to approve that. Hopefully another repository volunteer will see that and approve it.

benjaoming commented:
Thanks for the feedback! I've removed the emojis and added a few more words about what `hatch-fancy-pypi-readme` is. I think that the project is highly relevant to the "innovative" thinking of READMEs that champions the spirit of this Awesome List...

Robson commented:
I'm getting a bit confused as to why so much is being missed here, so I'll repeat:

Do you see that the review has three items? Only the third item is done.

Do you have an opinion on splitting this into two PRs? It's fine if you want to keep it as one PR, but it's unlikely to ever get approved.

benjaoming commented:
Thanks for the list and good luck!
